src/jrd/jrd.h

```diff
@@ -1078,18 +1078,28 @@ namespace Jrd {
 	class EngineCheckout
 	{
 	public:
-		EngineCheckout(thread_db* tdbb, const char* from, bool optional = false)
+		enum Type
+		{
+			REQUIRED,
+			OPTIONAL,
```

Comment:
Is `OPTIONAL` `#defined` in Windows?

Reply:
Yes, unfortunately (in many SDK headers).